shared/teams/new-team/wizard/new-team-info.tsx

```diff
@@ -93,25 +118,43 @@ const NewTeamInfo = () => {
       }}
       allowOverflow={true}
     >
-      <Kb.Box2 direction="vertical" fullWidth={true} style={styles.body} gap="tiny">
-        <Kb.LabeledInput
-          placeholder="Team name"
-          value={name}
-          onChangeText={setName}
-          maxLength={16}
-          autoFocus={true}
-        />
+      <Kb.Box2 direction="vertical" fullWidth={true} fullHeight={true} style={styles.body} gap="tiny">
+        {parentName ? (
+          <Kb.NewInput
+            autoFocus={true}
+            maxLength={16}
+            onChangeText={setName}
+            prefix={`${parentName}.`}
```

Comment:
Alignment of this is off on mobile, I wonder if something changed / if it's broken everywhere. I remember it working when implemented. :(
<img width="150" alt="image" src="https://user-images.githubusercontent.com/11968340/77958446-0b1e6b00-72a3-11ea-9c8d-cb9eab1c1e2d.png">